packages/@aws-cdk/aws-codedeploy/lambda-packages/ecs-deployment-provider/lib/is-complete.ts

```diff
@@ -0,0 +1,128 @@
+import { Logger } from '@aws-lambda-powertools/logger';
+import { CodeDeploy, DeploymentStatus, GetDeploymentOutput } from '@aws-sdk/client-codedeploy';
+
+const logger = new Logger({ serviceName: 'ecsDeploymentProviderIsComplete' });
+const codedeployClient = new CodeDeploy({});
+
+/**
+ * The request object that the custom resource lamba function receives from CloudFormation.
+ */
+export interface IsCompleteRequest {
+  /**
+   * The type of CloudFormation request (e.g. 'Create', 'Update', or 'Delete')
+   */
+  RequestType: string;
+
+  /**
+   * The physical resource id. Could be undefined for 'Create' events.
+   */
+  PhysicalResourceId: string;
+}
+
+/**
+ * The response object that the custom resource lambda function returns to CloudFormation.
+ */
+export interface IsCompleteResponse {
+  /**
+   * True iff the deployment is in a final state.
```

Comment:
```suggestion
   * True if the deployment is in a final state.
```

Reply:
This was intended to say `iff`...will replace with longer version of `if and only if`